helpers/github.ts

```diff
@@ -257,3 +257,71 @@ export async function getProjectUrls() {
 
   return projectUrls;
 }
+
+// Function to calculate total rewards from open issues
+export const calculateTotalRewards = async (issues: GitHubIssue[]) => {
```

Comment:
I don't think arrow functions should have passed the linter. Please be sure to run `yarn format`

Reply:
Yea, i ran `yarn format` and there was no warning on the arrow function use but i'd change it